test/example.c

```diff
@@ -230,12 +338,16 @@ void test_deflate(compr, comprLen)
 
     err = deflateEnd(&c_stream);
     CHECK_ERR(err, "deflateEnd");
+
+	{
```

Comment:
Why the scope?

Reply:
Oh. Thank you. This is left over from before the macro, when it was needed. I've gone through and removed them all.